web_external/templates/body/challengePhasesPage.jade

```diff
@@ -8,7 +8,7 @@ ul.c-phase-list
   each phase in phases
     li.c-phase-list-entry
       .c-challenge-phase-header
-      a.c-phase-link(c-phase-cid="#{phase.cid}")
+      a.c-phase-link(c-phase-cid="#{phase.cid}" href="#phase/#{phase.get('_id')}")
```

Comment:
Let's change this whole line while we're at it:

    a.c-phase-link(c-phase-cid=phase.cid, href="#phase/#{phase.id}")